invenio_app_rdm/theme/templates/invenio_app_rdm/header.html

```diff
@@ -0,0 +1,85 @@
+{# -*- coding: utf-8 -*-
```

Comment:
minor: 2 space indentation. If you are using VSCode: https://github.com/editorconfig/editorconfig-vscode is your friend.